website/content/docs/auth/jwt.mdx

```diff
@@ -0,0 +1,136 @@
+---
+layout: docs
+page_title: JWT - Auth Method
+description: >-
+  The JWT auth method allows authentication using signed JWTs
+---
+
+# JWT auth method
+
+@include 'x509-sha1-deprecation.mdx'
+
+The `jwt` auth method can be used to authenticate with Vault using JSON Web Tokens
+([JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)). This form of authentication
+is commonly used in Workload Identity Federation (WIF) workflows for machine-to-machine
+authentication.
+
+The auth method is configured to trust a JWT issuer. For example, this could be a
+cloud service provider that makes workload identity available in a virtual machine
+or CI provider that provides identity to jobs. It must be [configured](#jwt-verification)
+with public keys from the JWT issuer by either OIDC Discovery, a JSON Web Key Set (JWKS) URL,
+or locally-provided keys. A signed JWT from the configured issuer is required as the form
+of authentication. The auth method will cryptographically verify the signature using the
+issuer's public key. The verified JWT claims can be used to provide [further validation](#bound-claims)
+to gate authentication to Vault.
+
+## Authentication
+
+### Via the CLI
+
+```shell-session
+$ vault write auth/jwt/login role=demo jwt=...
+```
+
+The default path for the JWT authentication backend is `/jwt`, so if you're using the
+default backend, the command would be:
+
+```shell-session
+$ vault write auth/jwt/login role=demo jwt=...
+```
+
+If your JWT auth backend is using a different path, use that path.
+
+### Via the API
+
+The default endpoint is `auth/jwt/login`. If this auth method was enabled
+at a different path, use that value instead of `jwt`.
+
+```shell-session
+$ curl \
+    --request POST \
+    --data '{"jwt": "your_jwt", "role": "demo"}' \
+    http://127.0.0.1:8200/v1/auth/jwt/login
+```
+
+The response will contain a token at `auth.client_token`:
+
+```json
+{
+  "auth": {
+    "client_token": "hvs.abcd",
+    "accessor": "78e87a38-84ed-2692-538f-ca8b9f400ab3",
+    "policies": ["default"],
+    "metadata": {
+      "role": "demo"
+    },
+    "lease_duration": 2764800,
+    "renewable": true
+  }
+}
+```
+
+## Configuration
+
+Auth methods must be configured in advance before users or machines can
+authenticate. These steps are usually completed by an operator or configuration
+management tool.
+
+1. Enable the JWT auth method.
+
+   ```text
+   $ vault auth enable jwt
+   ```
+
+1. Use the `/config` endpoint to configure Vault. To support JWT roles, either local keys,
+   a JWKS URL, or an OIDC Discovery URL must be present. For an explanation of available
+   configuration options, see the [API documentation](/vault/api-docs/auth/jwt#jwt-verification)
+   section.
+
+   ```text
+   $ vault write auth/jwt/config \
+      oidc_discovery_url="https://MYDOMAIN.eu.auth0.com/"
+  ```
+
+1. Create a named role:
+
+   ```text
+   vault write auth/jwt/role/demo \
```

Comment:
Noticed some inconsistent use of `$`. I am not sure what the recommended format is? With or without `$` in front of commands?

Reply:
@schavis - Curious if you have a preference for this?

Reply:
Generally, we should use the `$` in example output to differentiate the command from the output text. But we should leave it off any example where we expect folks to copy/paste the command.